CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v2.0.1
```

Comment:
I think because this adds a new public method to `VariableBase`, and adds a new fields to the output UMM-Var records, this increment should be a minor version bump to v2.1.0, not a patch. (I'm also going to claim that the changes to `get_science_variables` are a bug-fix, and not a major change.)

Reply:
Also, I reckon we could probably release these changes as part of this PR. So that would mean adding the date and then updating `VERSION`.

Reply:
* [ ] update changelog to 2.1.0